tests/unit/records/test_records_facets.py

```diff
@@ -36,3 +36,9 @@ def test_range_author_count_filter():
             },
         },
     }
+
+
+def test_must_match_all_filter():
+    must_filter = must_match_all_filter('facet_author_name')
+    assert must_filter(['John Doe']).to_dict() == {'bool': {'must': [{'match': {'facet_author_name': 'John Doe'}}]}}
+    assert must_filter(['John Doe', 'John Doe 2']).to_dict() == {'bool': {'must': [{'match': {'facet_author_name': 'John Doe'}}, {'match': {'facet_author_name': 'John Doe 2'}}]}}
```

Comment:
are hard to read, maybe separate them in different variables

Reply:
Usually what we do is defining a variable `expected` that represents the test's oracle :crystal_ball: